src/components/search/LiveTVSearchResults.js

```diff
@@ -0,0 +1,195 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+
+import globalize from '../../scripts/globalize';
+import ServerConnections from '../ServerConnections';
+import SearchResultsRow from './SearchResultsRow';
+
+const CARD_OPTIONS = {
+    preferThumb: true,
+    inheritThumb: false,
+    showParentTitleOrTitle: true,
+    showTitle: false,
+    coverImage: true,
+    overlayMoreButton: true,
+    showAirTime: true,
+    showAirDateTime: true,
+    showChannelName: true
+};
+
+/*
+ * React component to display search result rows for live tv library search
+ */
+const LiveTVSearchResults = ({ serverId, parentId, collectionType, query }) => {
+    const [ movies, setMovies ] = useState([]);
+    const [ episodes, setEpisodes ] = useState([]);
+    const [ sports, setSports ] = useState([]);
+    const [ kids, setKids ] = useState([]);
+    const [ news, setNews ] = useState([]);
+    const [ programs, setPrograms ] = useState([]);
+    const [ videos, setVideos ] = useState([]);
+
+    const getDefaultParameters = () => ({
+        ParentId: parentId,
+        searchTerm: query,
+        Limit: 24,
+        Fields: 'PrimaryImageAspectRatio,CanDelete,BasicSyncInfo,MediaSourceCount',
+        Recursive: true,
+        EnableTotalRecordCount: false,
+        ImageTypeLimit: 1,
+        IncludePeople: false,
+        IncludeMedia: false,
+        IncludeGenres: false,
+        IncludeStudios: false,
+        IncludeArtists: false
+    });
+
+    // FIXME: This query does not support Live TV filters
```

Comment:
Just as a note to reviewers, this is the current behavior, just added a note that it is not working as expected and will require either server or frontend changes to fix.